deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml

```diff
@@ -16,8 +16,8 @@ spec:
   cpuRequest: ""
   # details about the kafka connection
   kafka:
-    # the Kafka service instance to be used. Example: kogito-kafka
-    instance: kogito-kafka
+    # let's leave this burden to KogitoInfra CR to deploy a new Kafka instance for us
```

Comment:
The comment seems to be misleading. From the CRD description above:
> UseKogitoInfra flags if the instance will use a provided
                    infrastructure by KogitoInfra CR. Setting this to true will configure
                    a KogitoInfra CR to install Kafka via Strimzi Operator.
From what I understand setting this property to false means that user will provide Kafka connection informations manually, not relying on KogitoInfra.

Reply:
I'll change the values. Actually, the operator will change the value to true if the user doesn't explicitly specify which external kafka or infinispan to use.